sites/partners/src/listings/PaperListingForm/UnitsSummaryForm.tsx

```diff
@@ -55,59 +71,57 @@ const UnitsSummaryForm = ({
       sqFeetMax: current?.sqFeetMax,
       minOccupancy: current?.minOccupancy,
       maxOccupancy: current?.maxOccupancy,
-      amiPercentage: current?.amiPercentage,
-      monthlyRentMin: current?.monthlyRentMin,
-      monthlyRentMax: current?.monthlyRentMax,
       totalCount: current?.totalCount,
       totalAvailable: current?.totalAvailable,
-      monthlyRentAsPercentOfIncome: current?.monthlyRentAsPercentOfIncome,
       priorityType: current?.priorityType,
-      rentType: getRentTypeFromUnitsSummary(current),
+      openWaitlist: current?.openWaitlist,
+      amiLevels: current?.amiLevels,
     },
   })
 
-  useEffect(() => {
-    setTempId(currentTempId)
-  }, [currentTempId])
+  const unitType = formWatch("unitType")
+
+  /**
+   * fetch form options
+   */
+  const { data: unitPriorities = [] } = useUnitPriorityList()
+  const { data: amis = [] } = useAmiChartList(jurisdiction)
 
   useEffect(() => {
-    const summary = summaries.filter((summary) => summary.tempId === tempId)[0]
+    const summary = summaries.find((summary) => summary.tempId === currentTempId)
     setCurrent(summary)
     reset({
       ...summary,
-      rentType: getRentTypeFromUnitsSummary(summary),
+      // @ts-ignore:next-line
+      unitType: summary?.unitType?.map((elem) => elem.id ?? elem.toString()),
```

Comment:
Hey @seanmalbert,
there is a bit of a type mismatch going on between how the backend wants to consume this data, and how the FieldGroup gives this data back.

basically the FieldGroup gives us an array strings (more specifically an array of unit type UUIDs), and wants to consume the data in that form too

the back end wants the data in the form of an IdDto (of the form { id: uuid } and provides it to us like that as well

this is kind of a work around for that so that 
1) if we have an id that we got from the DB then we use it,
2) if we don't we got this data from the FieldGroup then we do a toString() of the string so we can get around the typescript errors

we could also have done a typescript disable, but I opted for this way instead